test/e2e/mock-e2e-whitelisted.js

```diff
@@ -0,0 +1,192 @@
+  'https://swap.api.cx.metamask.io/networks/1/trades?destinationToken=0x6b175474e89094c44da98b954eedeac495271d0f&sourceToken=0x0000000000000000000000000000000000000000&sourceAmount=2000000000000000000&slippage=2&timeout=10000&walletAddress=0x5cfe73b6021e818b776b421b1c4db2474086a7e1',
+  'https://swap.api.cx.metamask.io/networks/1/trades?destinationToken=0x6b175474e89094c44da98b954eedeac495271d0f&sourceToken=0x0000000000000000000000000000000000000000&sourceAmount=3000000000000000&slippage=2&timeout=10000&walletAddress=0x5cfe73b6021e818b776b421b1c4db2474086a7e1',
+  'https://swap.api.cx.metamask.io/networks/1/trades?destinationToken=0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48&sourceToken=0x0000000000000000000000000000000000000000&sourceAmount=1000000000000000&slippage=2&timeout=10000&walletAddress=0x5cfe73b6021e818b776b421b1c4db2474086a7e1',
+  'https://swap.api.cx.metamask.io/networks/1/trades?destinationToken=0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48&sourceToken=0x0000000000000000000000000000000000000000&sourceAmount=50000000000000000000&slippage=2&timeout=10000&walletAddress=0x5cfe73b6021e818b776b421b1c4db2474086a7e1',
+  'https://swap.metaswap.codefi.network/networks/1/topAssets',
+  'https://tokens.api.cx.metamask.io/v3/assets?assetIds=solana%3A5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp%2Fslip44%3A501',
+  'https://tokens.api.cx.metamask.io/v3/assets?assetIds=solana%3A5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp%2Fslip44%3A501%2Csolana%3A5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp%2Ftoken%3A2RBko3xoz56aH69isQMUpzZd9NYHahhwC23A5F3Spkin',
+  'https://tokens.api.cx.metamask.io/v3/assets?assetIds=solana%3AEtWTRABZaYq6iMfeYKouRu166VU2xqa1%2Fslip44%3A501',
+  'https://tx-sentinel-ethereum-mainnet.api.cx.metamask.io/',
+  'https://tx-sentinel-ethereum-mainnet.api.cx.metamask.io/networks',
+  'https://unresponsive-rpc.test/',
+  'https://verify.walletconnect.com/e6360eaee594162688065f1c70c863b7',
+  'https://w.usabilla.com/ade130c1096e.js?lv=1',
+  'https://websites.cdn.getfeedback.com/embed/aaeNy60jTL/gf.js',
+  'https://www.4byte.directory/api/v1/signatures/?hex_signature=0x',
+  'https://www.4byte.directory/api/v1/signatures/?hex_signature=0x048226a0',
+  'https://www.4byte.directory/api/v1/signatures/?hex_signature=0x095ea7b3',
+  'https://www.4byte.directory/api/v1/signatures/?hex_signature=0x23b872dd',
+  'https://www.4byte.directory/api/v1/signatures/?hex_signature=0x2e1a7d4d',
+  'https://www.4byte.directory/api/v1/signatures/?hex_signature=0x39509351',
+  'https://www.4byte.directory/api/v1/signatures/?hex_signature=0x5f575529',
+  'https://www.4byte.directory/api/v1/signatures/?hex_signature=0x60806040',
+  'https://www.4byte.directory/api/v1/signatures/?hex_signature=0xd0e30db0',
+];
+
+const WHITE_LISTED_HOSTS = [
```

Comment:
I tried to be granular, adding each specific URL in the above list, but for the cases where there were a huge amount of requests to different URLS with the same domain, or cases where the URL changed for some reason (including random ids etc) then it has been added as a host in the below list.

Both lists should be reduced to 0 eventually